Microsoft.Toolkit.Uwp.SampleApp/readme.md

```diff
@@ -140,6 +140,8 @@ Select the category where you want your page to be listed and add the following
 ]
 ```
 
+> NOTE: If creating a new icon, follow the [Thumbnail Style Guide and templates](https://1drv.ms/f/s!AjZLNGpIZBbgr8c8Cok_8xAnZBRN_Q)
```

Comment:
Shouldn't these be in github rather than someone's onedrive?

Reply:
I decided to add them to onedrive to avoid someone overwriting them and to avoid increasing the size of the repo. I agree OneDrive is not ideal too. Open to suggestions

Reply:
What about a different repo that is linked to this one?

Reply:
Good idea, created a new repo